tools/debug-ui/src/Tier1View.tsx

```diff
@@ -0,0 +1,188 @@
+import { MouseEvent, ReactElement, useCallback, useMemo, useState } from "react";
+import { useQuery } from "react-query";
+import { fetchEpochInfo, fetchFullStatus, PeerInfoView } from "./api";
+import { formatDurationInMillis, formatTraffic, addDebugPortLink } from "./utils";
+import "./Tier1View.scss";
+
+type Tier1ViewProps = {
+    addr: string,
+};
+
+type PeerInfo = {
+    peer: PeerInfoView,
+    validator: string[],
+    routedValidator: string[],
+    statusClassName: string,
+}
+
+export const Tier1View = ({ addr }: Tier1ViewProps) => {
+    const { data: fullStatus, error: fullStatusError, isLoading: fullStatusLoading } =
+        useQuery(['fullStatus', addr], () => fetchFullStatus(addr));
+
+    if (fullStatusLoading) {
+        return <div>Loading...</div>;
+    }
+    if (fullStatusError) {
+        return <div className="network-info-view">
+            <div className="error">
+                {(fullStatusError as Error).stack}
+            </div>
+        </div>;
+    }
+    if (!fullStatus) {
+        return <div className="network-info-view">
+            <div className="error">No Data</div>
+        </div>;
+    }
+
+    let rendered = new Set();
+    const network_info = fullStatus.detailed_debug_status!.network_info;
+
+    let rows = new Array();
+
+    // tier1_connections contains TIER1 nodes we are currently connected to
+    network_info.tier1_connections!.forEach(function (peer) {
+        let account_key = "";
+        let proxies = new Array();
+
+        network_info.tier1_accounts_data!.forEach(account => {
+            if (account.peer_id == peer.peer_id) {
+                account_key = account.account_key;
+                account.proxies.forEach(proxy => {
+                    proxies.push(proxy.peer_id.substr(8, 5) + "...@" + proxy.addr);
```

Comment:
nit: Apparently substr is deprecated: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr and `substring(8, 13)` should be used.